front/src/components/Index.jsx

```diff
@@ -1,21 +1,24 @@
 import { Link } from "react-router-dom";
 import React from "react";
 import useQuestions from "components/Questions";
-// Currently displays all questions in database. Not practical if there are a lot of questions. Need to update so
-// that it displays the five latest polls.
+
 const Index = () => {
   const questions = useQuestions();
   const listQuestion = (item) => {
     const pollHeader = `- ${item.text}`;
-    const publishDate = `Published: ${item.pubDate}`;
+    //const publishDate = `Published: ${item.pubDate}`;
+    //assignment of a different key to each DOM element is a react requirement
     return (
-      <>
-        <h1 style={{ color: "black", fontSize: "16px" }}>
-          <Link to={`/detail/${item.id}`}>{pollHeader}</Link>
-        </h1>
-        <h2 style={{ color: "black", fontSize: "10px" }}>{publishDate}</h2>
-      </>
-    );
+
+      <h1 style={{ color: "black", fontSize: "16px" }}
+          key = {item.id}>
+        <Link to={`/detail/${item.id}`}>{pollHeader}</Link>
+      </h1>
+      /* <h1 style={{ color: "black", fontSize: "10px" }}
```

Comment:
We shouldn't ever check skeleton code into master if we can help it. Should we delete this?

Reply:
Agree. Will delete.